File: Java/src/lib/Human.java
package lib;

public class Human {
    private String country;
    private String city;
    private String address;
    private String name;
    private String religion;
    private int age;
    private double height;
    private double weight;
    private String gender;

    public void setAddress(String address) {
        this.address = address;
    }
    public String getAddress() {
        return address;
    }
    public void setCity(String city) {
        this.city = city;
    }
    public String getCity() {
        return city;
    }
    public void setCountry(String country) {
        this.country = country;
    }
    public String getCountry() {
        return country;
    }
    public Human() {

    }
    public Human(String name, String religion, int age, double height, double weight, String gender) {
        this.name = name;
        this.religion = religion;
        this.age = age;
        this.height = height;
        this.weight = weight;
        this.gender = gender;
    }
    public void setGender(String gender) {
        this.gender = gender;
    }
    public String getGender() {
        return gender;
    }
    public int getAge() {
        return age;
    }
    public void setAge(int age) {
        this.age = age;
    }
    public double getHeight() {
        return height;
    }
    public void setHeight(double height) {
        this.height = height;
    }
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public String getReligion() {
        return religion;
    }
    public void setReligion(String religion) {
        this.religion = religion;
    }
    public double getWeight() {
        return weight;
    }
    public void setWeight(double weight) {
        this.weight = weight;
    }

    @Override
    public String toString() {
         return "\nName: " + name + "\nReligion: " + religion
                + "\nAge: " + age + "\nGender: " + gender + "\nHeight: " + height + "\nWeight: " + weight
                + "\nAddress: " + country + ", " + city + ", " + address;
    }
}

File: Flask/Python/lib/Human.py
class Human:
    name = ""
    age = int()
    religion = ""
    height = float()
    weight = float()
    gender = ""

    def __init__(self, name, age, religion, height, weight, gender):
        self.name = name
        self.age = age
        self.religion = religion
        self.height = height
        self.weight = weight
        self.gender = gender

    def set_name(self, name):
        self.name = name

    def get_name(self):
        return self.name

    def set_age(self, age):
        self.age = age

    def get_age(self):
        return self.age

    def set_religion(self, religion):
        self.religion = religion

    def get_religion(self):
        return self.religion

    def set_height(self, height):
        self.height = height

    def get_height(self):
        return self.height

    def set_weight(self, weight):
        self.weight = weight

    def get_weight(self):
        return self.weight

    def set_gender(self, gender):
        self.gender = gender

    def get_gender(self):
        return self.gender

    def info(self):
        return "\n[Name: " + self.name + "]\n[Religion: " + self.religion + "]\n[Gender: " + self.gender + "]\n[Age: " + self.age + "]\n[Height: " + self.height + "]\n[Weight: " + self.weight + "]\n"
